ansible/roles/atmo-ssh-setup/tasks/ssh-setup.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Directly check linux distribution
```

Comment:
Good point, I will use setup instead.

Also, I didn't set the python interpreter for centos5 bc it didn't seem to be doing anything since `/usr/bin/python` is already in use. I didn't run into any problems because of this in vagrant VM or in my atmobeta testing.